lib/src/lib/table/components/table/table.component.html

```diff
@@ -46,17 +47,27 @@
         [maxColumnWidthFromTable]="maxColumnWidth"
         [customCellContent]="customCellContent"
         (headerClick)="handleHeaderClick($event)"
+        [data]="data"
       ></lab900-table-cell>
 
       <ng-content select=".lab900-table-cell"></ng-content>
 
+      <ng-container *ngIf="preFooterTitle">
+        <ng-container matColumnDef="preFooterTitle">
+          <td mat-footer-cell *matFooterCellDef [attr.colspan]="tableCells.length" class="lab900-table__pre-footer-title-cell">
+            {{ preFooterTitle }}
+          </td>
+        </ng-container>
+      </ng-container>
+
       <ng-container matColumnDef="actions-front">
         <th mat-header-cell *matHeaderCellDef></th>
         <td mat-cell *matCellDef="let element">
           <div style="width: 100%" fxLayout="row" fxLayoutGap="8" fxLayoutAlign="start center" *ngIf="tableActionsFront?.length">
             <lab900-action-button *ngFor="let action of tableActionsFront" [data]="element" [action]="action"></lab900-action-button>
           </div>
         </td>
+        <td mat-footer-cell *matFooterCellDef></td>
```

Comment:
shouldn't this have an `*ngIf`, to hide it in case there are no footers

Reply:
done